lib/common.js

```diff
@@ -25,6 +27,15 @@ export const supportedLocalesDetails = [
 ];
 export const supportedLocales = supportedLocalesDetails.map((l) => l.code);
 
+export const frauConfig = {
+	defaultLangTag: defaultLocale,
+	primary: supportedBaseLocales,
+	regions: supportedLocalesDetails.map(l => l.code.replace(/\w\w$/, r => r.toUpperCase())),
+	aliases: {
+		'en-AU': 'en-GB'
+	}
+};
```

Comment:
What's this stuff for?

Reply:
This is to replace `frau-locale-provider` stuff.

Reply:
Cool. Just wondering what value it has here vs. in `frau-localize-provider` directly and having `frau-locale-provider` import `defaultLocale`, `supportedBaseLocales` and `supportedLocalesDetails` from here. Just don't love this library needing to know about "frau" or "FRAs" at all.

Reply:
Good point! I'll make that change.